.github/workflows/cd.yml

```diff
@@ -48,6 +48,10 @@ jobs:
       - name: Set WGPU_NATIVE_VERSION
         run: echo "WGPU_NATIVE_VERSION=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
         shell: bash
+      - name: Setup caching
+        uses: Swatinem/rust-cache@v2
+        with:
+          key: build-${{ env.TARGET }}-${{ env.CACHE_SUFFIX }}
```

Comment:
AFAIK, the docker container can't benefit from this action - we would have to implement something else for that.
So, for now it's fine to leave caching out for the container builds.

Reply:
I just tried to see if there was a way to reduce the time it was taking to build on QEMU. I will revert them.

Reply:
removed.